webapp/locales/pt.json

```diff
@@ -41,193 +136,733 @@
     "following": "Seguindo",
     "shouted": "Aclamou",
     "commented": "Comentou",
-    "userAnonym": "Anonymous"
+    "userAnonym": "Anonymous",
+    "socialMedia": "Where else can I find",
+    "network": {
+      "title": "Network",
+      "following": "is following:",
+      "followingNobody": "follows nobody.",
+      "followedBy": "is followed by:",
+      "followedByNobody": "is not followed by anyone.",
+      "andMore": "and {number} more …"
+    },
+    "invites": {
+      "title": "Invite somebody to Human Connection!",
+      "description": "Enter thier email address for invitation.",
+      "emailPlaceholder": "Email to invite"
+    }
+  },
+  "notifications": {
+    "reason": {
+      "mentioned_in_post": "Mentioned you in a post …",
+      "mentioned_in_comment": "Mentioned you in a comment …",
+      "commented_on_post": "Commented on your post …"
+    },
+    "comment": "Comment"
+  },
+  "search": {
+    "placeholder": "Search",
+    "hint": "What are you searching for?",
+    "failed": "Nothing found"
   },
   "settings": {
-    "name": "Configurações",
+    "name": "Settings",
     "data": {
-      "name": "Seus dados",
-      "labelName": "Seu nome",
-      "namePlaceholder": "Anonymous",
-      "labelCity": "Sua cidade ou estado",
-      "labelBio": "Sobre você"
+      "name": "Your data",
+      "labelName": "Your Name",
+      "labelSlug": "Your unique user name",
+      "namePlaceholder": "Femanon Funny",
+      "labelCity": "Your City or Region",
+      "labelBio": "About You",
+      "success": "Your data was successfully updated!"
+    },
+    "email": {
+      "validation": {
+        "same-email": "This is your current email address"
+      },
+      "name": "Your email",
+      "labelEmail": "Change your email address",
+      "labelNewEmail": "New email Address",
+      "labelNonce": "Enter your code",
+      "success": "A new email address has been registered.",
+      "submitted": "An email to verify your address has been sent to <b>{email}</b>.",
+      "change-successful": "Your email address has been changed successfully.",
+      "verification-error": {
+        "message": "Your email could not be changed.",
+        "explanation": "This can have different causes:",
+        "reason": {
+          "invalid-nonce": "Is the confirmation code invalid?",
+          "no-email-request": "Are you certain that you requested a change of your email address?"
+        },
+        "support": "If the problem persists, please contact us by email at"
+      }
+    },
+    "validation": {
+      "slug": {
+        "regex": "Allowed characters are only lowercase letters, numbers, underscores and hyphens.",
+        "alreadyTaken": "This user name is already taken."
+      }
```

Comment:
This seems to be English, not Portuguese. :thinking:

Reply:
I'm still working on it ;)